<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorEmail</class>
 <widget class="QWidget" name="AuthorEmail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuthorEmail</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 24pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>  稿件作者使用</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edit_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>稿件名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>稿件内容：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edit_data">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="td">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;
background-color: rgb(0, 255, 127);</string>
   </property>
   <property name="text">
    <string>投递</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>投稿人：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edit_peo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
